resources/js/components/Nodes/NewNodeOverlay.vue

```diff
@@ -82,8 +82,9 @@ export default {
               title: edgeCluster,
               value: edgeCluster,
               options: config.status.hosts.map(host => {
+                const fb = host.fieldbus ? ` [${host.fieldbus}]` : "";
```

Comment:
Will this just print `[true]`? Surely we want to keep the arch?

Reply:
We can keep the arch as well if you like; it's a question of what information is helpful rather than confusing. The reason for pushing the arch in the first place was to identify Pis.